server/issue.go

```diff
@@ -113,26 +128,26 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 	reply := &model.Post{
 		// TODO: Why is this not created.Self?
 		Message:   fmt.Sprintf("Created a Jira issue %v/browse/%v", ji.GetURL(), created.Key),
-		ChannelId: post.ChannelId,
-		RootId:    create.PostId,
+		ChannelId: channelId,
+		RootId:    postId,
 		UserId:    mattermostUserId,
 	}
 	_, appErr = api.CreatePost(reply)
 	if appErr != nil {
 		return http.StatusInternalServerError,
-			errors.WithMessage(appErr, "failed to create notification post "+create.PostId)
+			errors.WithMessage(appErr, "failed to create notification post, postId: "+postId)
```

Comment:
It's really the root post Id is it not? The one we are replying to?  Maybe add the channel ID here to make this error message more useful.